src/Guide/Api/Guider.hs

```diff
@@ -0,0 +1,39 @@
+{- | Guider monad with Config to replace servant`s Handler. -}
+
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+
+module Guide.Api.Guider
+       ( Guider (..)
+       , GuiderServer
+       , guiderToHandler
+       ) where
+
+import Imports
+
+import Servant (Handler (..))
+import Servant.Server.Generic
+import Servant.Server.Internal.ServantErr (ServantErr)
+
+import Guide.Config (Config (..), def)
+
+
+-- | Guider type contains Config inside and replaces servant`s Handler type.
+newtype Guider a = Guider
```

Comment:
A comment should say _why_ something is being done, because _what_ is being done is usually the easy part (you can see it directly from the code). I.e.: why does `Guider` exist? What problem does it solve?